src/state/ThemeReducer.js

```diff
@@ -0,0 +1,15 @@
+import { defaultTheme, darkTheme } from '../state/ThemeContext';
+
+export default function reducer(state, action){
+    switch(action.theme){
+        case "DARK_THEME":{
+            return { theme: darkTheme };
+        }
+        case "DEFAULT_THEME":{
+            return { theme: defaultTheme };
+        }
```

Comment:
No need to create this case, since default option will trigger the default theme.